ansible/sandbox.yml

```diff
@@ -0,0 +1,103 @@
+---
+- hosts: all
+  gather_facts: no
+
+  tasks:
+  - name: install python 2
+    raw: test -e /usr/bin/python || (apt -y update && apt install -y python2.7)
+    become: yes
+
+- name: Provision ISIC Archive
+  hosts: sandbox
+
+  vars:
+    # TODO figure out how to programatically detect this:
```

Comment:
If you can get the info out of Terraform easily, you could use the EBS volume ID to identify the volume within `/dev/disk/by-id`.